### ISSUE TITLE
Add IFCOpenShell settings

### ISSUE THREAD

**Issue by jozefdobos** [3drepo/3drepogui#191]
Add settings for IFCOpenShell including:

- [x] Ability to switch IFCOpenShell on/off
- [x] Generation of IFCSpace spaces
- [ ] Level of detail setting with indication of progressively taking longer to process
- [x] Whatever else that can be set

**Comment by carmenfan** [comment]
requires https://github.com/3drepo/3drepobouncer/issues/161

**Comment by carmenfan** [comment]
LoD option is not available on Ifcopenshell, so further work needs to be done on IFCOpenShell to enable that.

Done otherwise. A new issue should be flagged once the IFCOpenShell work is complete on LoD
